scripts/pbs_cgroup.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+/opt/pbs/bin/qmgr -c "export hook pbs_cgroups application/x-config default" > pbs_cgroups.json
+
+tmp=$(mktemp)
+jq '.cgroup.devices.enabled = true' pbs_cgroups.json > $tmp && mv $tmp pbs_cgroups.json
+jq '.cgroup.devices.allow = [ "b *:* rwm","c 10:* rwm","c 4:* rwm","c 5:* rwm","c 1:* rwm","c 7:* rwm",["nvidia-uvm", "rwm"]]' pbs_cgroups.json > $tmp && mv $tmp pbs_cgroups.json
+
+/opt/pbs/bin/qmgr -c "import hook pbs_cgroups application/x-config default pbs_cgroups.json"
+
+/opt/pbs/bin/qmgr -c "export hook pbs_cgroups application/x-python default" > pbs_cgroups.py
+
+cat > pbs_cgroups_py.patch << EOF
```

Comment:
Can this patch be ported in the official PBS OpenSource release ?
What will happen in case of a version upgrade ?

Reply:
Yes, will check to see if this bug is corrected in the current PBS opensource repo, if not I will submit a PR.